SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -45,7 +46,8 @@ InterestFactory::InterestFactory(AActor* InActor, const FClassInfo& InInfo, USpa
 void InterestFactory::CreateAndCacheInterestState(USpatialClassInfoManager* ClassInfoManager)
 {
 	ClientCheckoutRadiusConstraint = CreateClientCheckoutRadiusConstraint(ClassInfoManager);
-	ClientResultType = CreateClientResultType(ClassInfoManager);
+	ClientInterestResultType = CreateClientInterestResultType(ClassInfoManager);
```

Comment:
ClientNonAuthResultType?

Reply:
Used these names to better align with the concepts of Interest and Authority in spatial, but I guess both are technically interest, so will change if you feel strongly

Reply:
Actually I think I agree with this because they're both interest- will change